WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -9,7 +9,7 @@
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.Wallets.WalletSettingsView">
   <c:ContentArea Title="{Binding Title}"
-                 Caption="Manage your wallet's settings"
+                 Caption=""
```

Comment:
I suppose we can remove the property altogether?

Reply:
As I mentioned in the description I didn't remove it because I want to keep space under the dialog's title.
Is there a better way to achieve that?

Reply:
Just add a Margin on the child `StackPanel`

Reply:
Done